tests/testthat/test-convert_quarter_to_year.R

```diff
@@ -0,0 +1,17 @@
+test_that("properly converts a single unique quarter string to a year integer", {
+  expect_equal(convert_quarter_to_year("2019Q4"), 2019L)
```

Comment:
Do we want to check that the Q2 quarters convert correctly as well?

Reply:
sure...
```suggestion
  expect_equal(convert_quarter_to_year("2019Q2"), 2019L)
  expect_equal(convert_quarter_to_year("2019Q4"), 2019L)
```